Signal/src/contact/OWSContactsManager.h

```diff
@@ -51,8 +51,13 @@ extern NSString *const OWSContactsManagerSignalAccountsDidChangeNotification;
 
 #pragma mark - Util
 
+// returns a saved contact name if available, else a saved profile name, else nil
+- (nullable NSString *)nonPhoneNameForPhoneIdentifier:(NSString *_Nullable)recipientId;
```

Comment:
I started down a different path and failed to remove the unused code after a refactor. Good catch!